kubernetes/resource_kubernetes_service.go

```diff
@@ -65,6 +65,11 @@ func resourceKubernetesService() *schema.Resource {
 							Elem:        &schema.Schema{Type: schema.TypeString},
 							Set:         schema.HashString,
 						},
+						"external_traffic_policy": {
+							Type:        schema.TypeString,
+							Description: "Only applies to `type = NodePort` or `type = LoadBalancer`. Defines the policy for external traffic. Can be Cluster (default) or Local. Local adds proxy rules only for pods that exist on the same node.",
+							Optional:    true,
```

Comment:
@dnikolai It would be great to rebase this for good measure and add :+1: 

```
ValidateFunc: validation.StringInSlice([]string{"Cluster", "Local"}, false),
Default: "Cluster",
"
here